src/algebraic_geometry/presheafed_space.lean

```diff
@@ -132,13 +151,20 @@ lemma id_c (X : PresheafedSpace C) :
   ((𝟙 X) : X ⟶ X).c = eq_to_hom (presheaf.pushforward.id_eq X.presheaf).symm := rfl
 
 @[simp] lemma id_c_app (X : PresheafedSpace C) (U) :
-  ((𝟙 X) : X ⟶ X).c.app U = eq_to_hom (by { induction U using opposite.rec, cases U, refl }) :=
+  ((𝟙 X) : X ⟶ X).c.app U = X.presheaf.map
+    (eq_to_hom (by { induction U using opposite.rec, cases U, refl })) :=
 by { induction U using opposite.rec, cases U, simp only [id_c], dsimp, simp, }
 
 @[simp] lemma comp_base {X Y Z : PresheafedSpace C} (f : X ⟶ Y) (g : Y ⟶ Z) :
   (f ≫ g).base = f.base ≫ g.base := rfl
 
-@[simp] lemma comp_c_app {X Y Z : PresheafedSpace C} (α : X ⟶ Y) (β : Y ⟶ Z) (U) :
+/--
+The `reassoc` attribute was added despite the LHS not being a composition of two homs, since
+1. This allows us to rewrite in the opposite direction easier.
+2. Sometimes rewriting `comp_c_app` wouldn't work because of dependent type issues, but
+  `erw comp_c_app_assoc` sometimes solves this problem.
+-/
```

Comment:
Could you please rewrite this docstring so that it is helpful when read outside the context of this file?
Something like:
```suggestion
-- The `reassoc` attribute was added despite the LHS not being a composition of two homs,
-- for the reasons explained in the docstring.
/-- Sometimes rewriting with `comp_c_app` doesn't work because of dependent type issues.
In that case, `erw comp_c_app_assoc` might make progress.
The lemma `comp_c_app_assoc` is also better suited for rewrites in the opposite direction. -/
```